internal/api/core/user.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/edulinq/autograder/internal/model"
 )
 
+var SERVER_USER_INFO_ROW_COLUMNS []string = []string{"email", "name", "server-role", "course-name", "course-role"}
```

Comment:
I am very skeptical of moving this to a higher dependency location (where it used to be in model).
However, with the changes below, this may not even need to exist
(or should just be a constant in cmd/).